.github/pull_request_template.md

```diff
@@ -25,6 +25,16 @@
 
 - ...
 
+## 🚀 Has this modified a publishable library?
+
+<!-- AT LEAST ONE box must be checked. Do not delete this section! see libs/README.md for info on publishing -->
+
+This PR:
+
+- [ ] modifies an **npm**-publishable library and **I have added a changeset file(s)**
+- [ ] does not modify any publishable libraries
+- [ ] I am unsure / need advice
```

Comment:
This is now out of sync with the # repo. Do we want them to match?

Reply:
@CiaranMn I was thinking of this: https://github.com/blockprotocol/blockprotocol/tree/main/libs/%40blockprotocol/type-system/crate - maybe not qualifying.

Reply:
And I also understand with https://github.com/blockprotocol/blockprotocol/pull/1127 we will have a block.

Reply:
Fair enough – I have modified it

Reply:
> https://github.com/blockprotocol/blockprotocol/tree/main/libs/%40blockprotocol/type-system/crate - maybe not qualifying.

The crate isn't publishable at the moment (we just import it from git), but it would be nice to change that at some point :)